### ISSUE TITLE
Rule cognitive-complexity panics in presence of functions w/o body

### ISSUE THREAD

**Issue by chavacava** [mgechev/revive#451]
**Describe the bug**
`cognitive-complexity` rule panics when analysing functions without body 

**To Reproduce**
1. I updated revive `go get -u github.com/mgechev/revive`
2. I run it with the following flags & configuration file:

```shell
# flags

revive -config defaults.toml target.go
```

```toml
# config file
ignoreGeneratedHeader = false
severity = "warning"
confidence = 0.8
errorCode = 0
warningCode = 0

[rule.cognitive-complexity]
  arguments =[1]
```

```go
// target.go

package target

func myFuncWithoutBody()
```

**Expected behavior**
`revive` must not panic, the rule must calculate a cognitive complexity of 0 (zero) for the function

**Additional context**
[Declaring functions without body is accepted by GO](https://golang.org/ref/spec#Function_declarations)
